modules/proc-providing-feedback-on-redhat-documentation.adoc

```diff
@@ -0,0 +1,46 @@
+[preface]
+[id="proc-providing-feedback-on-redhat-documentation"]
+= Providing feedback on Red Hat documentation
+
+We appreciate your feedback on our documentation.
+To provide feedback, you can highlight the text in a document and add comments.
+
+This section explains how to submit feedback.
+
+.Prerequisites
+
+* You are logged in to the Red Hat Customer Portal.
+* In the Red Hat Customer Portal, view the document in *Multi-page HTML* format.
```

Comment:
Good point, @oraNod 

You can also go the other way:

```suggestion
* You are logged in to the Red Hat Customer Portal.
* You are viewing the document in *Multi-page HTML* format.
```

Reply:
@inoxx03 "You are viewing" doesn't sound right to me. Imperative voice for prereqs imho.

Reply:
@oraNod I see your point. Imperatives for prereq's work well, too, I think, as long as we use them consistently.

Reply:
Cool. I mean different styles are fine and all but that "You have/are x" phrasing is too verbose for my liking. In general it feels harder to parse for readers and awkward for writers. 

No offense to anyone's work but that phrasing has come out of nowhere in a lot of RH docs lately and I simply don't agree that we should use it. Imperative structure is concise, authoritative, and lends well to parallel construction. Those are cornerstones of quality technical content imho.